### ISSUE TITLE
[maven] Updating to old version

### ISSUE THREAD

**Issue by mmaeller** [renovatebot/config-help#505]
### Which Renovate are you using? x the right box:

- [x] Renovate Open Source CLI
- [ ] WhiteSource Renovate App
- [ ] Renovate Pro
- [ ] Other (please write below)

### Which platform are you using? x the right box:

- [ ] GitHub.com
- [ ] GitHub Enterprise
- [ ] GitLab.com
- [x] GitLab self-hosted
- [ ] Bitbucket Server
- [ ] Bitbucket Cloud
- [ ] Azure DevOps

### What would you like to do?

When renovate runs it wants to update the `spring-cloud-dependencies` from `Hoxton.SR1` to `Hoxton.RELEASE`, which is older than SR1 (see [Maven-Central](https://mvnrepository.com/artifact/org.springframework.cloud/spring-cloud-dependencies)).
Can we somehow configure renovate that it picks the newer version? So it should pick Hoxton.SR2, when it's available and currently not Hoxton.RELEASE.
We are using renovate 19.90.8.

```
  <dependencyManagement>
    <dependencies>
      <dependency>
        <groupId>org.springframework.cloud</groupId>
        <artifactId>spring-cloud-dependencies</artifactId>
        <version>Hoxton.SR1</version>
        <type>pom</type>
        <scope>import</scope>
      </dependency>
    </dependencies>
  </dependencyManagement>
```

**Comment by Manummental** [comment]
@mmaeller  yo utilizo @renovatebot

**Comment by rarkins** [comment]
Please upgrade to the latest version - I believe there was a fix for this recently.

**Comment by mmaeller** [comment]
@rarkins Unfortunately it didn't help. Still wants to update to the older "RELEASE" version.

**Comment by rarkins** [comment]
Sorry! Fix was just applied in 19.107.3

**Comment by mmaeller** [comment]
Thank you, works now
